tests/tt_metal/tt_metal/test_kernels/misc/brisc_print.cpp

```diff
@@ -20,6 +20,14 @@ void kernel_main() {
     }
     cb_push_back(cb_id_in0, 1);
 
+    cb_reserve_back(tt::CB::c_in2, 1);
+    auto float_ptr = reinterpret_cast<float *>(get_write_ptr(tt::CB::c_in2));
+    float float_base = 3.14159274101257324219;
```

Comment:
M_PI (or maybe a modern equivalent)?

Reply:
Ended up removing this constant in the end